### PR TITLE
fix: Disable possibility of vault creation with chain name or an url

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM on web and extension

**Review by konradodo — APPROVED** [review]

File: src/composables/accounts.ts [hunk 1]
```diff
@@ -11,14 +11,17 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
...
+    return accounts.value.map((acc) => ({
+      text: withName ? getAccountNameToDisplay(acc) : acc.address,
       value: acc.address,
       address: acc.address,
-    })),
-  );
+      listOnlyText: getAccountNameToDisplay(acc),
+    }));
+  }
+  const accountsSelectOptions = computed(() => prepareAccountsSelectOptions(true));
```
Comment (peronczyk) [inline src/composables/accounts.ts:22]: I suggest to add empty line before this one.

File: src/composables/accounts.ts [hunk 2]
```diff
@@ -11,14 +11,17 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
...
 
+  const plainAccountSelectOptions = computed(() => prepareAccountsSelectOptions());
```
Comment (peronczyk) [inline src/composables/accounts.ts:24]: I suggest to rename this to `accountSelectOptionsPlain` to follow the same naming as the `accountSelectOptions`. Also I suggest to add empty line after this one.

File: src/popup/components/Modals/MultisigVaultCreate.vue [hunk 3]
```diff
@@ -266,7 +266,8 @@ export default defineComponent({
     const canCreateMultisig = computed(
       () => (
         signers.value.length >= MULTISIG_VAULT_MIN_NUM_OF_SIGNERS
-        && isValidSigners.value
+            && isValidSigners.value
+            && !(root as any).$validator.errors.items?.length
```
Comment (peronczyk) [inline src/popup/components/Modals/MultisigVaultCreate.vue:270]: Wrong indentation.

File: src/composables/accounts.ts [hunk 4]
```diff
@@ -11,14 +11,18 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
...
-    (): IFormSelectOption[] => accounts.value.map((acc) => ({
-      text: getAccountNameToDisplay(acc),
+  function prepareAccountsSelectOptions(withName = false): IFormSelectOption[] {
+    return accounts.value.map((acc) => ({
+      text: withName ? getAccountNameToDisplay(acc) : acc.address,
       value: acc.address,
       address: acc.address,
-    })),
-  );
+      listOnlyText: getAccountNameToDisplay(acc),
```
Comment (peronczyk) [inline src/composables/accounts.ts:19]: In my opinion this is bad to have a possibility of 2 properties with the same values (when the `withName` is set to `true`.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/composables/accounts.ts [hunk 5]
```diff
@@ -11,14 +11,18 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
...
+    }));
+  }
+
+  const accountsSelectOptions = computed(() => prepareAccountsSelectOptions(true));
 
+  const accountSelectOptionsPlain = computed(() => prepareAccountsSelectOptions());
```
Comment (peronczyk) [inline src/composables/accounts.ts:25]: I suggest to add empty line after this one.

**Review by peronczyk — APPROVED** [review]

**Comment by csiarab** [comment]
Lgtm

**Comment by Liubov-crypto** [comment]
LGTM on web and extension

**Pull request merged**